parlai/zoo/model_list.py

```diff
@@ -2910,4 +2910,28 @@
             [GPT2ComboSearchQuery]: ParlAI: A new software platform for dialog research. Image credit: AI Research, ML Applications
         """,
     },
+    {
+        "title": "FairScore Demographic Perturber",
+        "id": "perturber",
```

Comment:
I'd give this a more unique ID

Reply:
i think you need to change the ID to what the directory is in the zoo

so here it'll need to be `fairscore` -- that appears to be the failing test